### ISSUE TITLE
problems about data set partitioning

### ISSUE THREAD

**Issue by LucasWang16** [v1xerunt/StageNet#2]
Hi,
I used the benchmark tools to complete data set partitioning of MIMIC-iii. However, There is no validation set in it. And the csv file in your ./data dictionary is just for seveal examples. Can you release the complete patitioning csv file?
Thanks,
Lucas

**Comment by v1xerunt** [comment]
Hi, have you run mimic3models/split_train_val.py files in the benchmark code? I think that's the code to split valid dataset.

**Comment by LucasWang16** [comment]
Thank you for your help. Your work is great. Besides,  I find that there is several variants of this method, decay-LSTM and Health-ATM for comparison in your paper. In the description, they are different from the original models because of the particularity of this task itself. Can I get these variants in your project? Or I have to implement these comparative experiments according to the descriptions in your paper.
Thanks!

**Comment by v1xerunt** [comment]
Yes, they do require some additional features. For fairness, we use the same training features for all baselines and our model.

1. Health-ATM requires additional target-aware attention and a time-aware convnet. For target-aware attention, as we wrote in the paper, since our task don’t have a specific target disease embedding to guide the attention, we remove the target-aware attention mechanism from Health-ATM. For time-aware convnet, you can use train_time/valid_time/test_time matrix in the train.py code. These are the time intervals between visits.

2. deacy-LSTM requires measurement time intervals for each feature. In train.py, we store this in train_interval/valid_interval/test_interval.

Maybe you need to adjust the matrix shape for your code.
